### PR TITLE
Added requirement filtering

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 8e87e5c4d51e42ee801c9af9b0d9f9e2e96321e9:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: fixtures/node_annotate.py [hunk 1]
```diff
@@ -46,6 +49,8 @@ def pytest_configure(config):
 def pytest_addoption(parser):
     group = parser.getgroup('cfme')
     group.addoption('--tier', type=int, action='append', help='only run tests of the given tiers')
+    group.addoption('--requirement', type=str, action='append',
+        help='only run tests of the given requirement')
```
Comment (jkrocil) [inline fixtures/node_annotate.py]: Just a minor thing here to avoid possible confusion - `only run tests of the given requirements`.

Comment (psav) [inline fixtures/node_annotate.py]: Thanks!

**Comment by dajoRH** [comment]
Lint report for commit 02f9d262ce63870f44ceaaad5f71117b3c4524b0:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**